flink-table/flink-table-planner/src/main/java/org/apache/calcite/sql2rel/SqlToRelConverter.java

```diff
@@ -2419,23 +2425,15 @@ protected void convertFrom(
                 return;
 
             case TABLE_REF:
-                call = (SqlCall) from;
-                convertIdentifier(bb, call.operand(0), null, call.operand(1), null);
+                convertTableRef(bb, from, false);
```

Comment:
According to the contract, please wrap every change with 
```
// ----- FLINK MODIFICATION BEGIN -----
// ----- FLINK MODIFICATION END -----
```
and update the line number in the javadoc

Reply:
Done!